### ISSUE TITLE
Dont set reply contact for chatrooms

### ISSUE THREAD

**Issue by GoogleCodeExporter** [baluragala/gtalksms#117]
```
The problem is it also opens a main chat window whose only purpose 
is to display "Reply contact is Suzy (+11231231233)".  This is 
unnecessary with a group chat window, and frustrating to have to close 
each time. 

Can you change it so when I am using separated chats, NOTHING goes to 
the main GTalkSMS window? 

The same should apply for "SMS delivered".  This should go into the 
group chat window itself, not a separate one. 

I hope you're able to agree to change all the above ASAP :) 
```

Original issue reported on code.google.com by `earthlin...@googlemail.com` on 6 Mar 2011 at 8:48

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 7 Mar 2011 at 7:41
- Changed title: **Dont set reply contact for chatrooms**
- Changed state: **WaitingForRelease**
- Added labels: **Priority-Low**, **Type-Enhancement**
- Removed labels: **Priority-Medium**, **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in v2.0
```

Original comment by `Florent....@gmail.com` on 8 Mar 2011 at 7:34
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 25 Aug 2011 at 7:50
- Changed state: **Deployed**
